programs/bench.h

```diff
@@ -16,8 +19,25 @@
 #define ZSTD_STATIC_LINKING_ONLY   /* ZSTD_compressionParameters */
 #include "zstd.h"     /* ZSTD_compressionParameters */
 
+typedef struct {
+    size_t cSize;
+    double cSpeed;   /* bytes / sec */
+    double dSpeed;
+} BMK_result_t;
```

Comment:
What happens if there is an error ?
Or : is there a guarantee that `BMK_benchMem()` will never fail ?